Source/Core/VideoCommon/VertexShaderGen.h

```diff
@@ -37,9 +37,12 @@ struct vertex_shader_uid_data
 	u32 numColorChans        : 2;
 	u32 dualTexTrans_enabled : 1;
 	u32 pixel_lighting       : 1;
-	u32 pad                  : 1;
+	u32 msaa : 1;
 
+	u32 ssaa : 1;
 	u32 texMtxInfo_n_projection : 16; // Stored separately to guarantee that the texMtxInfo struct is 8 bits wide
```

Comment:
Please try to not move such fields. They should aligned as good as possible with the source way to get a faster UID generator.